# TEST FILE

# Initial standard deviations
pitch_roll_std: [0.1, 0.1]

# Camera calibration
distortion_coeffs: [0.0, 0.0, 0.0, 0.0]
distortion_model: radtan
resolution: [640, 480]
intrinsics: [640, 480, 320 , 240]
T_imu_cam:
 - [0.41802349, -0.63293951,  0.6516471,  -0.056874]
 - [0.75952742, -0.15001891, -0.63293951,  0.35648 ]
 - [0.49837182,  0.75952742,  0.41802349, -0.84165 ]
 - [0.0,         0.0,         0.0,         1.0     ]

# IMU noise statistics
accelerometer_noise_density: 0.01   
accelerometer_random_walk:   0.001
gyroscope_noise_density: 0.01
gyroscope_random_walk:   0.001

# State options
enable_camera_extrinsic_calibration: true
enable_camera_intrinsic_calibration: true
gravity: 9.81
num_clones: 10
num_persistent_features: 50

# Other options
persistent_feature_init_delay: 1.0
state_transition_order: 1

# Tracker
equalization_method: none
optical_flow_pyramid_levels: 3
feature_detector: fast
grid_x_size: 6
grid_y_size: 4
min_feature_pixel_distance: 5
min_features: 20
max_features: 150
fast_threshold: 5
shi_tomasi_quality_level: 0.05

# Track Manager
max_track_length: 25

# Logger level [0: Full, 1: INFO, 2: WARN, 3: ERR, 4: INACTIVE]
logger_level: 4
